src/WebJobs.Script.Host/Program.cs

```diff
@@ -25,7 +26,9 @@ public static void Main(string[] args)
                 RootScriptPath = rootPath
             };
 
-            ScriptHostManager scriptHostManager = new ScriptHostManager(config);
```

Comment:
Can you just have a default parameter for this like we have for ScriptHostFactory? That way no changes are needed to this file.